### ISSUE TITLE
pna.nextTick is not a function

### ISSUE THREAD

**Issue by mafintosh** [npm/npm#20848]
#### I'm opening this issue because:

  - [x] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] npm is producing incorrect or undesirable behavior.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

Running

``` sh
npm install array-uniq
```

Crashes with the following error:

```
npm http fetch GET 200 https://registry.npmjs.org/array-uniq 29ms (from cache)
npm ERR! asyncWrite is not a function
npm WARN enoent ENOENT: no such file or directory, open '/tmp/tmpdir/dp1WvI/package.json'
npm WARN dp1WvI No description
npm WARN dp1WvI No repository field.
npm WARN dp1WvI No README data
npm WARN dp1WvI No license field.

npm ERR! pna.nextTick is not a function
npm http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/audits/quick 838ms

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/maf/.npm/_logs/2018-05-31T13_24_05_356Z-debug.log
```

Running `npm version`

```js
{ npm: '6.1.0',
  ares: '1.14.0',
  cldr: '33.0',
  http_parser: '2.8.0',
  icu: '61.1',
  modules: '64',
  napi: '3',
  nghttp2: '1.29.0',
  node: '10.3.0',
  openssl: '1.1.0h',
  tz: '2018c',
  unicode: '10.0',
  uv: '1.20.3',
  v8: '6.6.346.32-node.9',
  zlib: '1.2.11' }
```

On Arch Linux

**Comment by mafintosh** [comment]
Turns out this was caused by the dependency flattening in 10.3.0. After manually clearing the node_modules first it worked.

**Comment by realJoshByrnes** [comment]
I've noticed a few people who had this issue (that linked here) were using 'n' node version manager

$ n stable
$ npm install -g npm
$ n latest

Worked for me. No longer getting any issues.

**Comment by VRedondo-zz** [comment]
I have the same issues and I after run `npm install -g npm` it stills returns
asyncWrite is not a function

**Comment by marcusadair** [comment]
A slight variation on instructions from @ozjd worked for me (in case it's helpful for those who come here).

$ n lts
$ npm install -g npm
$ n stable
$ npx create-react-app ... # <-- no errors now

Presumably it's related to use of n for node version management.

**Comment by nonkr** [comment]
In node 10.4.0 and 10.3.0, I still got the errors, so I fallback to 10.2.1:blush:

```
$ n 10.2.1
```

**Comment by legodude17** [comment]
@nonkr This is actually a bug with `n`, so please update that.

**Comment by jmilloy** [comment]
I was able to fix the issue by forcing readable-stream@2.3.0.
